vnext/Shared/FuseboxInspectorThread.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <dispatchQueue/dispatchQueue.h>
+
+namespace Microsoft::ReactNative {
+
+class FuseboxInspectorThread final {
+ public:
+  static Mso::DispatchQueue &Instance() {
+    static Mso::DispatchQueue queue = Mso::DispatchQueue::MakeSerialQueue();
```

Comment:
I used a SerialQueue so I didn't have to worry about tearing anything down, maybe that's not really true though.